assemblies/data-access-plugin/src/main/resources/settings.xml

```diff
@@ -19,6 +19,14 @@
   <!-- default view acls for user or role -->
   <data-access-default-view-acls>31</data-access-default-view-acls>
 
+  <!-- The number of threads used for preloading a user's available data sources, 
+     a process which is started upon login.
+     Available data sources are displayed in the "Manage Data sources" dialog and in 
+     applications such as Analyzer and Interactive Reporting. When the preloading 
+     process is not complete, the user has to wait when accessing these.
+     The default number of threads is the number of CPU cores in the system -->
+  <data-access-datasource-load-threads>4</data-access-datasource-load-threads>
```

Comment:
What I meant was that the commented element should either have an empty value or a reasonable **example value**, such as `4`, instead of the phrase which was previously there, describing the default. I did not mean that the element should not be commented, especially, because the proposed **example value** is a kind of a wild guess, which might not be adequate for all systems.

Trying to state some ground:
1. The default value is that which is used when the configuration is either absent or empty.
2. The value of "as many threads as cores" is a _dynamic_ value, not a static/constant one.
3. It is common for "dynamic values" to be associated with a keyword, e.g. `auto`. Then, the default (absent or empty) could be, for example, `4`, and the user would still have a way to configure the special, automatic behavior.
4.  The default (dynamic or constant) should be safe for all cases and be the one most cases would use.

Concluding:

**Solution 1**
If `4` fits item 4), then the default value should be changed to `4`, and a keyword such as `auto` would be developed to allow for "as many threads as cores" (assuming it's actually an important value to support).

**Solution 2**
Otherwise, if `4` does not fit item 4), but the value "as many threads as cores" does, then we're all good. The dynamic value does not need an actual keyword value, as it is referenced by the absent/empty value. However, then, if the element keeps its example value of `4`, it should be commented out, and let the **actual default value** surface. Alternatively, commented or not, clear the element's value (set it to empty).

Makes sense?